js/client.js

```diff
@@ -709,7 +709,17 @@ function toId() {
 		connect: function () {
 			if (this.down) return;
 
-			if (Config.server.banned || (Config.bannedHosts && Config.bannedHosts.indexOf(Config.server.host) >= 0)) {
+			var serverIsBanned = Config.bannedHosts && Config.bannedHosts.includes(Config.server.host);
```

Comment:
```suggestion
			var serverIsBanned = false;
```

This is redundant with the for-loop.